freud/environment.pyx

```diff
@@ -532,11 +532,20 @@ cdef class EnvironmentCluster(_MatchEnv):
                 global_search=False):
         r"""Determine clusters of particles with matching environments.
 
-        In general, it is recommended to specify a number of neighbors rather
-        than just a distance cutoff as part of your neighbor querying when
-        performing this computation. Using a distance cutoff alone could easily
-        lead to situations where a point doesn't match a cluster because a
-        required neighbor is just outside the cutoff.
+        Using a distance cutoff for :code:`'env_neighbors'` would
+        lead to situations where the :code:`'cluster_environments'`
+        contain different number of particles.
+
+        .. warning::
+
+            All vectors of :code:`'cluster_environments'` are defined with
+            respect to the origin. Zero vectors are only used to pad the cluster
```

Comment:
The center of the motif is moved to the origin I think. Otherwise the zero vectors would mean particles at the origin. See in this [comment](https://github.com/glotzerlab/freud/pull/979#issuecomment-1164555080) where Erin replied to me

Reply:
In that case, I think it would be more appropriate to say something along the lines of "All vectors of :code:`cluster_environments` are defined relative to the average position of the particles in the cluster.".

Reply:
Yeah that makes sense. I see why this can be confusing now. But I'll just check with @erteich again to make sure.

Reply:
Hi all, cluster_environments is a list of environments that were found via clustering, meaning that each environment in cluster_environments corresponds to the environment of a single "cluster" of particles with similar environments (ie neighborhoods). The way the code works is that cluster_environments[i]  is actually the average environment taken over all the environments of particles in cluster i. So if cluster 0 consists of 4 particles (indexed k1-k4) which have similar environments defined by their 6 nearest neighbors, then cluster_environments[0] is a set of 6 vectors, where each vector is the average taken over the corresponding matching vector in the environments of particles k1-k4. As an example, cluster_environments[0][0] will be a vector, calculated as the following average: (k1_environment[0]+k2_environment[0]+k3_environment[0]+k4_environment[0])/4. Each vector in every particle's environment is defined _with respect to the particle_, which is what enables these averages to take place across different particle environments. Put another way, one could imagine placing each particle at the origin when defining the vectors that make up its environment. So I would say "cluster_environments[i] contains the average environment of the cluster of particles that is indexed by i. Each vector in cluster_environments[i] is defined with respect to an imaginary particle sitting at the origin." Does this make sense and seem okay? Feel free to change the wording, especially of the word "imaginary." But I can't think of a clearer way of saying it- presumably the user will understand how environments are generally defined throughout the code, and so saying that this variable contains "average environments" won't be too confusing?

The averaging process can be seen in the code, if you are curious or if looking at the code itself would be clearer than my explanation: line 626-631 of [environment.pyx](https://github.com/glotzerlab/freud/blob/master/freud/environment.pyx) shows that cluster_environments returns the set of environments produced by getClusterEnvironments() in [MatchEnv.h](https://github.com/glotzerlab/freud/blob/master/cpp/environment/MatchEnv.h) (defined lines 283-286). That function just returns the object m_cluster_environments, which is populated in lines 656-660 of [MatchEnv.cc](https://github.com/glotzerlab/freud/blob/master/cpp/environment/MatchEnv.cc) from the object cluster_env, which is computed in lines 632-639 of MatchEnv.cc via the method getAvgEnv (defined lines 192-238). Wow now that I actually summarized it I'm not sure if this is clearer than my explanation... but just in case you want to see how it is working under the hood, that is what's happening!

Reply:
Thank you @erteich! I'll include the definition of cluster_environment in the documentation as well. This is crystal clear. I guess in my original change what was confusing, or what I think @tommy-waltmann was confused about, was that the "origin" could mean the origin of the system. But your example is great, and it definitely helps to explain how the cluster_environments are computed.